sdks/java/extensions/google-cloud-platform-core/src/main/java/org/apache/beam/sdk/extensions/gcp/auth/NoopCredentialFactory.java

```diff
@@ -39,8 +38,12 @@ public static NoopCredentialFactory fromOptions(PipelineOptions options) {
     return INSTANCE;
   }
 
+  public static NoopCredentialFactory getInstance() {
+    return INSTANCE;
+  }
+
```

Comment:
Couldn't you just use `NoopCredentialFactory.fromOptions` function that is right above? Or am I missing something?

Reply:
Reverted and changed to using `fromOptions`